uwblueprint/feeding-canadian-kids
# 54 reset password functionality

## Zenhub ticket link

[Onboarding: Reset Password Functionality](https://github.com/uwblueprint/feeding-canadian-kids/issues/54)



## Implementation description
* Merges `a4bello/reset-password-functionality-update` in, which introduces a new mutation for resetting password
* Connects Reset Password UI to the backend mutation so that users can reset their password
* Implements redirection upon success, and error toast upon error



## Steps to test
1. Login with a valid password and email
2. Check the database for your user's ID, and navigate to /{id}/reset-password
3. Verify that the UI matches designs
4. Enter a new password that is at least 8 characters long, and click "Reset"
5. If successful, you should be redirected to the dashboard. In this case, try logging out and logging back in with your new password.
6. If an error occurs, an error toast should appear.



## What should reviewers focus on?
* Checking that the password actually successfully resets


## Checklist
